deployment/terraform/scripts/rds_shared_snapshot.sh

```
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+function error_exit() {
+  echo "$1" 1>&2
+  exit 1
+}
+
+function check_deps() {
+  test -f $(which jq) || error_exit "jq command not detected in path, please install it"
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)